twig/filesystem.py

```diff
@@ -1,8 +1,11 @@
 from pathlib import Path
+from typing import Generator
 
 SOURCES_GLOB = "**/*.vy"
 
 
-def collect_sources(path, glob=SOURCES_GLOB):
+def collect_sources(
+    path: Path, glob: str = SOURCES_GLOB
+) -> Generator[Path, None, None]:
```

Comment:
For generators we have often used `Iterable[Path]` since it is more concise and I think mypy is fine with it.